src/release/breaking-changes/index.md

```diff
@@ -39,6 +39,7 @@ release, and listed in alphabetical order:
 
 ### Released in Flutter 3.16
 
+* [Migrate RawKeyEvent/RawKeyboard system to KeyEvent/HardwareKeyboard system][]
```

Comment:
Should this be under 'Not yet released to stable'?

Reply:
Yes, please list this under the header "Not yet released to stable".